### PR TITLE
add SimpleATsit5

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
This was kept separate to keep SimpleTsit5 as.. well... simple. The adaptivity is the PI-controlled adaptivity. Some more of those constants can be propagated up to keyword and default arguments. both SimpleTsit5 and SimpleATsit5 are on the DiffEqBase API so they can be plopped right in where Tsit5 is. It doesn't have an initial `dt` choice. It doesn't handle `saveat` though that would be essential to most use cases so that should get added. 

Since this doesn't need to get broadcasted for abstract array handling (GPUArrays), we can make these instead use a loop over the tableau part and generic to the RK coefficients, then add a version with Vern7/8/9 coeffs. Note that when doing those it should disable the FSAL part. That would give a simple version of all of the basic RK methods.

**Comment by ChrisRackauckas** [comment]
@Datseris

File: src/tsit5/atsit5.jl [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+      integ.u_modified=false
+    else
+      k1 .= k7
+    end
+
+
+    EEst = Inf
+
+    @inbounds while EEst>1
+      dt < 1e-14 && error("dt<dtmin")
```
Comment (Datseris) [inline src/tsit5/atsit5.jl]: Typo or intended ?

Comment (ChrisRackauckas) [reply src/tsit5/atsit5.jl]: intended

**Pull request merged**